ydb/_topic_reader/topic_reader_asyncio.py

```diff
@@ -171,14 +171,14 @@ async def flush(self):
 
         use asyncio.wait_for for wait with timeout.
         """
-        raise NotImplementedError()
+        await self._reconnector.flush()
 
-    async def close(self):
+    async def close(self, flush: bool = False):
```

Comment:
flush must be True by default (and False while close called from __del__).

Main idea: 
When a person write:
msg = reader.receive_message()
...
reader.commit(msg)
reader.close()

- expected behaviour: the message will be committed to a server.

From other side on __del__ is not guaranteed way to do something, it will collect garbarage only.